doc/Release.md

```diff
@@ -0,0 +1,70 @@
+2. It ensures the release is build exactly according to the source as defined by the branch that was used for the release.
+   This includes the workflow file defining how the release is done itself.
+3. Our release should achieve [SLSA build level 3](https://slsa.dev/spec/v1.0/levels#build-l3) as it [runs on GitHub](https://slsa.dev/spec/v1.0/threats). Some more properties [are thus achieved](https://slsa.dev/spec/v1.0/threats). 
+
+That said, here are some caveasts:
+* None of our dependencies has any SLSA verification yet (as of Dec 2023) and thus we (cannot) verify any of our own supply chain.
+  That said, this is mitigated by the fact that the project has not that many dependencies at all.
+
+For more information [see the corresponding issue](https://github.com/PrivateBin/PrivateBin/issues/1169) and [the GitHub workflow file](/.github/workflows/release.yml).
+
+#### How to verify
+
+Git signature for verifying the reproducibility and that a maintainer in posession with that PGP private key created the release with that content:
+```
+$ gpg2 --verify PrivateBin-1.2.1.tar.gz.asc # TODO: update for v1.6.2
+gpg: assuming signed data in 'PrivateBin-1.2.1.tar.gz'
+gpg: Signature made Sat Aug 11 22:38:29 2018 CEST
+gpg:                using RSA key 0xE11B7950E9E183DB
+gpg: Good signature from "PrivateBin release (solely used for signing releases)" [<trust level here>]
+gpg:                 aka "PrivateBin code-sign (solely used for code signing) <git@privatebin.net>" [<trust level here>]
+Primary key fingerprint: 28CA 7C96 4938 EA5C 1481  D42A E11B 7950 E9E1 83DB
+```
+
+For a more step-by-step guide in detail [see this FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ#how-can-i-securely-clonedownload-your-project).
+
+## Reproducible builds
+
+Since a long time, we signed release `.tar.gz` and `.zip` files with personal PGP keys confirming the authenticity of the fact that the release has been build by a PrivateBin maintainer.
+
+This uses traditional [PGP](https://en.wikipedia.org/wiki/Pretty_Good_Privacy) signatures.
+
+### How to verify
+
+Git signature for verifying the reproducibility and that a maintainer in posession with that PGP private key created the release with that content:
+```
+$ gpg2 --verify PrivateBin-1.2.1.tar.gz.asc # TODO: update for v1.6.2
+gpg: assuming signed data in 'PrivateBin-1.2.1.tar.gz'
+gpg: Signature made Sat Aug 11 22:38:29 2018 CEST
+gpg:                using RSA key 0xE11B7950E9E183DB
+gpg: Good signature from "PrivateBin release (solely used for signing releases)" [<trust level here>]
+gpg:                 aka "PrivateBin code-sign (solely used for code signing) <git@privatebin.net>" [<trust level here>]
+Primary key fingerprint: 28CA 7C96 4938 EA5C 1481  D42A E11B 7950 E9E1 83DB
+```
+
+For a more step-by-step guide in detail [see this FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ#how-can-i-securely-clonedownload-your-project).
```

Comment:
I am open whether to move the instructions there into this doc and keep only a reference there. (On the other hand I also like that the instructions are kept minimal here.) Also SLSA should probably be mentioned in the FAQ.

Reply:
I de-duped it and am happy to for now keep the details in the FAQ. I added a link to the SLSA verifier - unfortunately it is not yet available in packages of most distros, so installing it poses it's own verification issues. Early days, in the mean time we still got gpg which is widely supported.

In detail, I...:
- removed duplicate content
- clarified what is signed and since which release (1.0)
- updated verification sample output (also in wiki) using the output of https://github.com/PrivateBin/docker-nginx-fpm-alpine/actions/runs/7228527399/job/19698112056#step:7:156
- Corrected release process - we trigger the release on the push of the tag, not a branch (fixed point in time instead of a moving target)

Reply:
> Edit: I also removed the section about dependency verification - because we vendor them in (included in the repo) we do actually provide full verification via git commit signatures when they are updated by devs and baking them into the signed release archives.

Ah well but we cannot verify the SLSA properties, as they don't have such. Aka we cannot verify e.g. the jQuery (minified) build is built out of the jquery source code located somewhere…